src/event-actions/all.coffee

```diff
@@ -98,36 +112,38 @@ module.exports =
       when "unlabeled"
         msg += " #{sender.login} removed label: \"#{data.label.name}\" "
 
-    callback msg + "- #{issue.html_url}"
+    callback msg
 
-  issue_comment: (data, callback) ->
+  issue_comment: (data, callback, adapter) ->
     issue = data.issue
     comment = data.comment
     repo = data.repository
+    repo_link = formatUrl adapter, repo.html_url, repo.name
+    comment_link = formatUrl adapter, comment.html_url, "#{issue_pull} \##{issue.number}"
```

Comment:
I don't believe you have to escape the `#` symbol in CoffeeScript. The code you have here, per coffeescript.org, produces:

```js
issue_pull + " \#" + issue.number;
```